api_docs/playlists.md

```diff
@@ -291,8 +291,101 @@ playlist[priority] | The <strong>priority</strong> of the playlist related with
 
 ## Smart Ordering
 
-<p>Playlist smart ordering allows you to change the order of its videos by sort options. The videos added or updated into a playlist with sort options are sorted automatically following that sort option</p>
+<p>Playlist smart ordering allows you to change the order of its videos by sort options. The videos added or updated into a playlist with sort options are sorted automatically following that sort option<p>
 
 Parameter | Function | Type
 --------- | -------- | ----
 playlist[sort_options] | An array of sort options. Only two levels of sorting are accepted, and videos are sorted first by first level, and second by second level | Array
+
+<p>Every <strong>sort_option</strong> has three fields:</p>
+
+<p>
+  <ul>
+    <li><strong>direction:</strong> The direction to sort the results. It accepts two values: <strong>asc</strong> (ascending) and <strong>desc</strong> (descending) order.</li>
```

Comment:
let's define (ascending) as "(ascending - lowest to highest)" and (descending) as "(descending - highest to lowest)"